server/app/controllers/FeatureFlagOverrideController.java

```diff
@@ -0,0 +1,32 @@
+package controllers;
+
+import annotations.FeatureFlagOverrides;
+import auth.Authorizers;
+import javax.inject.Inject;
+import org.pac4j.play.java.Secure;
+import play.mvc.Http.Request;
+import play.mvc.Result;
+
+/**
+ * Allows for overriding of feature flags by the CiviForm Admin via HTTP request.
+ */
+public class FeatureFlagOverrideController extends CiviFormController {
```

Comment:
How about putting the controllers / views in a `/dev` directory, similar to what we do for the `/dev/seed/` routes?

Reply:
done